### ISSUE TITLE
feat: new boxo example running kubo-clone programmatically

### ISSUE THREAD

**Issue by SgtPooki** [ipfs/go-libipfs#464]
There are currently only 3 examples: https://github.com/ipfs/boxo/blob/main/examples/README.md

It would be nice to have an example showing how to kick off a "kubo daemon" programmatically.

This would unblock https://github.com/SgtPooki/ipfs-desktop-wails/issues/12

**Comment by SgtPooki** [comment]
There is an existing example of using kubo as a library at https://github.com/ipfs/kubo/tree/master/docs/examples/kubo-as-a-library, but how does that change when using boxo?
